dashboard/src/components/entitypage/elements/MetricsDropdown.js

```diff
@@ -57,54 +71,97 @@ const MetricsDropdown = ({ type, name, version, timeRange }) => {
+                  <div className={classes.titleBar}>
+                    <div className={classes.graphTitle}>
+                      <Typography variant="h6">
+                        Throughput (req/{stepRange})
+                      </Typography>
+                    </div>
+
+                    <div className={classes.aggDropdown}>
+                      <TimeDropdown />
+                      <AggregateDropdown graph={0} />
+                    </div>
+                  </div>
+
                   <QueryDropdown
                     query={`rate(test_counter{feature="${name} ${version}",status="success"}[${step}])`}
                     type={type}
                     name={name}
                     query_type={"latency"}
+                    aggregate={aggregates[0]}
                   />
-                  <Typography> Average Latency (ms)</Typography>
+                  <div className={classes.titleBar}>
+                    <div className={classes.graphTitle}>
+                      <Typography variant="h6">Average Latency (ms)</Typography>
+                    </div>
+
+                    <div className={classes.aggDropdown}>
+                      <TimeDropdown />
+                      <AggregateDropdown graph={1} />
+                    </div>
+                  </div>
+
                   <QueryDropdown
-                    query={`rate(test_duration_seconds_sum{feature="${name} ${version}"}[${step}])/rate(test_duration_seconds_count{feature="${name} ${version}"}[${step}])`}
+                    query={`rate(test_duration_seconds_${aggregates[0]}{feature="${name} ${version}"}[${step}])/rate(test_duration_seconds_count{feature="${name} ${version}"}[${step}])`}
                     type={type}
                     name={name}
                     query_type={"count"}
+                    aggregate={aggregates[0]}
                   />
                 </div>
               ) : (
                 <div>
-                  <Typography>Throughput (rows/{stepRange})</Typography>
+                  <div className={classes.titleBar}>
+                    <div className={classes.graphTitle}>
+                      <Typography variant="h6">
+                        Throughput (rows/{stepRange})
+                      </Typography>
+                    </div>
+
+                    <div className={classes.aggDropdown}>
+                      <TimeDropdown />
+                      <AggregateDropdown graph={1} />
+                    </div>
+                  </div>
+
                   <QueryDropdown
                     query={`rate(test_counter{feature="${name} ${version}",status="row serving"}[${step}])`}
                     type={type}
                     name={name}
                     query_type={"latency"}
+                    aggregate={aggregates[1]}
                   />
                 </div>
               )}
 
-              <Typography>Errors per {stepRange}</Typography>
+              <div className={classes.titleBar}>
```

Comment:
This breaks out of my screen
![image](https://user-images.githubusercontent.com/64661186/152076281-fe8d6b76-41c7-4014-b306-f368c07a9878.png)

Reply:
Fixed on latest push